bugbug/repository.py

```diff
@@ -24,8 +25,10 @@
 db.register(COMMITS_DB, 'https://www.dropbox.com/s/mz3afgncx0siijc/commits.json.xz?dl=1', 'v1')
 
 COMPONENTS = {}
```

Comment:
Rename this to `path_to_component`, so the rest of the code gets more readable (since you are adding many variables with similar names).